config/renovate/renovate.json

```diff
@@ -78,10 +78,28 @@
       }
     ]
   },
+  "customManagers": [
+    {
+      "customType": "regex",
+      "description": "Fedora updates from a stable source",
+      "fileMatch": ["(^|/|\\.)([Dd]ocker|[Cc]ontainer)file$", "(^|/)([Dd]ocker|[Cc]ontainer)file[^/]*$"],
+      "matchStrings": ["quay.io/fedora/fedora:(?<currentValue>.[0-9]+)"],
```

Comment:
These fedora images can come from other registries, especially `registry.fedoraproject.org`, and there are different variants, like: `fedora-minimal` and etc.

Reply:
Thats a good point, thank you. I updated the PR to match these cases as well, here is an example of renovate creating [2 separate MRs](https://gitlab.cee.redhat.com/lnovotna/msaas/-/merge_requests) for 2 fedora sources.